providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/grpc/GrpcConnector.java

```diff
@@ -61,9 +63,9 @@ public GrpcConnector(final FlagdOptions options, final Cache cache, Consumer<Pro
         this.stateConsumer = stateConsumer;
     }
 
-
     /**
      * Initialize the gRPC stream.
+     *
      * @throws RuntimeException if the connection cannot be established.
      */
     public void initialize() throws RuntimeException {
```

Comment:
I'd still like to see something like this done, if possible. I think that was the meaning of the 3rd checkbox [here](https://github.com/open-feature/java-sdk-contrib/issues/390).

Reply:
I am revisiting the loigc. will see if we can get rid of dummy call